Automattic/jetpack
# Add/open ai useselect

Fixes #28389

## Proposed changes:
 - It works with unsaved posts,
 - It handles multiple AI paragraph blocks at the same time
 - Updates nb of charcaters left dynamically
 - Takes into accounts categories and tags
 - Code is not spaghetti-based

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Sandbox a WPCOM site and push this to your favorite sandbox with 
```
$> ~/public_html/bin/jetpack-downloader test jetpack add/open-ai-useselect
```
- create a new post
- add a title only
- create AI paragraph
- it should tell you it misses characters. Enter some and voilà.
- add another AI parapgrah blocks behind it, it should say it needs to wait for the previous block to fini
<img width="200" alt="Screenshot 2023-01-20 at 20 15 13" src="https://user-images.githubusercontent.com/790558/213786250-2983bc3c-f823-4d90-a8ab-04ea2c0d29d9.png">


- Open the console and play with it, you can add/remove categories and tags, and it should send the right prompts.
